### PR TITLE
Fix encoding issues with utils.search_bytes_in_file.

### PR DESCRIPTION
**Pull request by oliverchang**
Renamed from search_string_in_file.

### CODE REVIEW

File: src/python/base/utils.py [hunk 1]
```diff
@@ -626,15 +626,21 @@ def restart_machine():
...
+  old_pos = file_handle.tell()
+  file_handle.seek(0, os.SEEK_SET)
+
+  try:
+    # TODO(aarya): This is too brittle and will fail if we have a very large
+    # line.
+    for line in file_handle:
+      if search_bytes in line:
+        return True
+  finally:
```
Comment (inferno-chromium) [inline src/python/base/utils.py]: do we need to log_error or log_warn on failures ?

File: src/python/base/utils.py [hunk 2]
```diff
@@ -626,15 +626,21 @@ def restart_machine():
...
+def search_bytes_in_file(search_bytes, file_handle):
+  """Helper to search for bytes in a large binary file without memory
   issues.
   """
-  # TODO(aarya): This is too brittle and will fail if we have a very large line.
-  search_string = search_string.encode('utf-8')
-  for line in file_handle:
-    if search_string in line:
-      return True
+  old_pos = file_handle.tell()
```
Comment (inferno-chromium) [inline src/python/base/utils.py]: Do we have usecase where file_handle position is not zero. anyway good to do this.

File: src/python/base/utils.py (hunk 1 above)
Comment (oliverchang) [reply src/python/base/utils.py]: The finally isn't for exception handling. It's just to make sure we reset the location.

File: src/python/base/utils.py (hunk 2 above)
Comment (oliverchang) [reply src/python/base/utils.py]: Not that I know of. I'll leave this here just in case.

**Comment by inferno-chromium** [comment]
I think this is causing

```
UnsupportedOperation: seek
at search_bytes_in_file (/mnt/scratch0/clusterfuzz/src/python/base/utils.py:633)
at is_fuzz_target_local (/mnt/scratch0/clusterfuzz/src/python/bot/fuzzers/utils.py:69)
at _get_fuzz_targets_from_archive (/mnt/scratch0/clusterfuzz/src/python/build_management/build_manager.py:577)
```
on chromium instance.

**Pull request merged**